packages/auth/index.extension.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Core functionality shared by all clients
+export * from './src';
+
+import { ClientPlatform } from './src/core/util/version';
+
+import { browserSessionPersistence } from './src/platform_browser/persistence/session_storage';
```

Comment:
Ack. Removed.

Reply:
FWIW - you can't use localStorage by default, since it doesn't exist in service workers, but you _can_ use `chrome.storage`, which does have [session storage](https://developer.chrome.com/docs/extensions/reference/api/storage#property-session).